### PR TITLE
Refactor herfin

### PR DESCRIPTION
**Pull request by lyc0603**

### CODE REVIEW

File: environ/utils/data_loader.py [hunk 1]
```diff
@@ -37,3 +37,25 @@ def load_data(
     df_merged.rename(columns=rename_dict, inplace=True)
 
     return panel_main.merge(df_merged[data_col], **kwargs)
+
+
+def load_data_herf(
```
Comment (xujiahuayz) [inline environ/utils/data_loader.py:42]: This looks almost exactly like the merge main panel function - is it necessary to create another function? If you create one function for one specific case, it completely defeat the purpose of a function (which is to be called and reused)

File: environ/constants.py
Comment (xujiahuayz) [inline environ/constants.py:1]: It’s time to clean up this file - please remove old unnecessary constants

File: environ/utils/data_loader.py (hunk 1 above)
Comment (lyc0603) [reply environ/utils/data_loader.py]: Make sense.

File: environ/constants.py
Comment (lyc0603) [reply environ/constants.py:1]: Understood.

**Pull request merged**